src/cpu/reorder/cpu_reorder_regular_f32_bf16.cpp

```diff
@@ -34,6 +35,8 @@ const impl_list_map_t &regular_f32_bf16_impl_list_map() {
             DNNL_NON_X64_ONLY(REG_SR_BIDIR(f32, any, bf16, nChw16c))
             DNNL_NON_X64_ONLY(REG_SR_BIDIR(f32, any, bf16, nCdhw16c))
 
+            DNNL_AARCH64_ONLY(CPU_REORDER_INSTANCE(aarch64::jit_uni_reorder_t))
```

Comment:
(random spot): @kawakami-k  Could you please take a look since the changes touch JIT implementation?

Reply:
@milpuz01 
My recommendation is written in https://github.com/kawakami-k/oneDNN/commit/7d171af565a6e777ffdc59ed2cc5a555d3e346fb

Thank you!